service/controller/resource/servicemonitor/service/simple-service.go

```diff
@@ -0,0 +1,119 @@
+package service
+
+import (
+	"fmt"
+	"strings"
+
+	promv1 "github.com/coreos/prometheus-operator/pkg/apis/monitoring/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/giantswarm/prometheus-meta-operator/service/key"
+)
+
+// SimpleService returns a ServiceMonitor that selects services labelled with
+// the "giantswarm.io/monitoring" label.
+func SimpleService(cluster metav1.Object) *promv1.ServiceMonitor {
+	return &promv1.ServiceMonitor{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "simple-service",
+			Namespace: key.Namespace(cluster),
+			Labels: map[string]string{
+				key.ClusterIDKey(): key.ClusterID(cluster),
+			},
+		},
+		Spec: promv1.ServiceMonitorSpec{
+			Selector: metav1.LabelSelector{
+				MatchLabels: map[string]string{
+					"giantswarm.io/monitoring": "true",
+				},
+			},
+			NamespaceSelector: promv1.NamespaceSelector{
+				Any: true,
+			},
+			Endpoints: []promv1.Endpoint{
+				{
+					Port:   "https",
+					Scheme: "https",
+					RelabelConfigs: []*promv1.RelabelConfig{
+						{
+							SourceLabels: []string{"__meta_kubernetes_namespace"},
+							TargetLabel:  "namespace",
+						},
+						{
+							SourceLabels: []string{"__meta_kubernetes_pod_name"},
+							TargetLabel:  "pod", // TODO or "pod_name" ??
+						},
+						// Preserve selected service discovery metadata
+						// TODO if this works, add "namespace" here and remove above
+						{
+							Action: "labelmap",
+							Regex: fmt.Sprintf(
+								"__meta_kubernetes_(%s)",
+								strings.Join([]string{
+									"node_name", "service_name", "pod_name", "pod_ip",
+									"pod_container_name", "pod_ready", "pod_phase",
+									"pod_node_name", "endpoints_name",
+								}, "|"),
+							),
+						},
+						// Default metrics path
+						{
+							Replacement: "/metrics",
+							TargetLabel: "__tmp_metrics_path",
+						},
+						// Extract default port from __address__ before rewriting it.
+						{
+							Regex:        "(.+):(\\d+)",
```

Comment:
I thought ports could the name of the port like "metrics" as well

Reply:
Good question. When they're specified by user, definitely, here the default `__address__` comes from Prometheus service discovery, so I'm not sure, but yeah good point, I better check it out.